scripts/kind-e2e/e2e.sh

```diff
@@ -95,117 +66,13 @@ function setup_custom_cni(){
             echo "Applying weave network in to cluster${i}..."
             kubectl --context=cluster${i} apply -f "https://cloud.weave.works/k8s/net?k8s-version=$(kubectl version | base64 | tr -d '\n')&env.IPALLOC_RANGE=${POD_CIDR[cluster${i}]}"
             echo "Waiting for weave-net pods to be ready cluster${i}..."
-            kubectl --context=cluster${i} wait --for=condition=Ready pods -l name=weave-net -n kube-system --timeout=700s
+            kubectl --context=cluster${i} wait --for=condition=Ready pods -l name=weave-net -n kube-system --timeout=360s
```

Comment:
I might should do this in a different change. @manosnoam, you were the one asking to make this timeout longer ages ago - do you still think it needs to be so long?

Reply:
Thanks for the clarification @manosnoam! I added a FIXME inline to reduce the timeout, but didn't change it in this commit to avoid confusion and maintain the scope of the commit.